selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -198,7 +198,7 @@ def test_fw_query_timing(self):
         'nissan': 0.9,
         'subaru': 0.1,
         'tesla': 0.2,
-        'toyota': 0.7,
+        'toyota': 1.0,
```

Comment:
we can bring this down to only 0.1 seconds, but I just want to see a few examples with all queries on this ECU